development-docs/DEV_GUIDE.md

```diff
@@ -34,15 +50,239 @@ structure after everything is said and done should look like `/Library/Java/Java
 doing that run the command at the beginning again and this should link the file and allow you to use maven with OpenJDK
 version 11.
 
-## Build from source
+## Build and deploy from source
+
+To build the operator from source the code needs to be compiled into a container image and placed
+in a location accessible to the Kubernetes/OpenShift nodes. The easiest way to make your personal build
+accessible, is to place it on [Quay.io](https://quay.io), [Docker Hub](https://hub.docker.com/) or another 
+container registry of your choice. Other build options (including options for limited or no network access) 
+are available in the sections below this quick start guide.
+
+1. If you don't have one already, create an account for your container registry. Then log your local
+   Docker client into the registry using:
+
+        docker login <registry_name>
+
+   You can use `quay.io` for the registry name if using that registry or omit the registry name for Dockerhub. This 
+   command sets the credentials for the `docker push` registry target.
+
+
+2. Make sure that the `DOCKER_ORG` and `DOCKER_REGISTRY` environment variables are set to the same value as your
+   repository on the container registry, and the container registry you are using. For Docker Hub  and Quay the repository is your 
+   username.
+
+        export DOCKER_ORG=repository
```

Comment:
Curious question ... does this default to `$USER' as in operators?

Reply:
Yes, it's set to $USER in the Makefile.docker if not set